SpinCore_pp/config_parser_fn.py

```diff
@@ -0,0 +1,90 @@
+from pyspecdata import *
+import configparser
+
+class configuration(object):
+    # this registers the type, the pretty case we want, the section, and
+    # whether or not we can assume a default value
+    registered_params = {
+        "amplitude":(float, "acq_params", None),
+        "deadtime_us":(float, "acq_params", None),
+        "tau_us":(float, "acq_params", None),
+        "deblank_us":(float, "acq_params", None),
+        "SW_kHz":(float, "acq_params", None),
+        "acq_time_ms":(float, "acq_params", None),
+        "carrierFreq_MHz":(float, "acq_params", None),
+        "startconstant":(float, "acq_params", None),
+        "stopconstant":(float, "acq_params", None),
+        "p90_us":(float, "acq_params", None),
+        "gamma_eff_MHz_G":(float, "acq_params", 0.00425),
+        "mw_freqs":(float,"acq_params",9.821e9), # JF asks -- what is this in contrast to the uw_...?
```

Comment:
sorry yes so the og parameter name we had given it is mw_freqs (standing for Microwave frequency) and is used in calculating the ppt in the field sweep processing (need to divide by). So rather than hard coding what we found with the spyder, it just pulls from the dip calculated in the dip lock function. EQUIVALENT TO nu_ESR but calculated from dip lock

Reply:
I'm saying that we can't have both `mw_freqs` and `uw_dip_center_GHz` --> we should pick one.  We should then be sure that you can still run the whole protocol after deleting the redundant one.  Also, whichever you pick, the units should be included in the name.